.travis.yml

```diff
@@ -45,8 +45,10 @@ script:
 before_cache:
   - find $HOME/.sbt -name "*.lock" | xargs rm
   - find $HOME/.ivy2/cache -name "ivydata-*.properties" | xargs rm
+  - find $HOME/.coursier -name "*.lock" | xargs rm
 cache:
   directories:
     - $HOME/.ivy2/cache
     - $HOME/.sbt
     - $HOME/.sdkman
+    - $HOME/.coursier
```

Comment:
```suggestion
    - $HOME/.cache/coursier
```

https://get-coursier.io/docs/cache says:

> - on Linux, `~/.cache/coursier/v1`. This also applies to Linux-based CI environments, and FreeBSD too.
>
> ...
> The default location above was set up in coursier 1.0.0-RC12-1 (released on 2017/10/31). The default cache path of former versions was `~/.coursier/cache/v1`, both on Linux and OS X.
>
> In order not to re-download all artifacts again, the first version of coursier run on a system, either former or newer, sets the location for newer versions.

In other words, the location of the Coursier cache changes depending on if you've ever ran older coursier or not.

Reply:
gah

Reply:
> depending on if you've ever ran older coursier or not.

How does Coursier determine that?  

Travis doesn't cache `~/.coursier` or `~/.cache/coursier` until the user says so.

Reply:
I guess coursier will use `.coursier` if it exists, and `.cache/coursier` otherwise. In the case of travis, both don't exist we didn't cache so far, so it's using the `.cache/coursier`. I checked that with my https://github.com/lrytz/travis-test repo.